website/src/utils/priceUtils.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 Enactic, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export const formatPrice = (price: number): string => {
+  return price.toLocaleString('ja-JP', {
+    style: 'currency',
+    currency: 'JPY'
+  });
+};
+
+export const calculateTotalCost = <T extends { unitPrice: number; quantity: number; }>(
+  components: T[]
+): number => {
+  return components.reduce(
+    (total, component) => (total + (component.unitPrice * component.quantity)),0);
```

Comment:
```suggestion
    (total, component) => (total + (component.unitPrice * component.quantity)), 0);
```

BTW, this code is difficult to understand that `0` is the second argument of `reduce()`...

Reply:
fix: [e4e7b3f](https://github.com/enactic/openarm/pull/140/commits/e4e7b3f96e4200164d9745ffa9c672116cefb01a) I simplified the function logic.